### PR TITLE
feat: profile view

### PR DESCRIPTION
**Pull request by momentiris**
**Ticket:** [#129](https://trello.com/c/O6VGxSSF/129-min-profil)
**Intent:**
Add profile index view with data from cv.

Also made small adjustments to other components in accordance with the warnings we got from react lint.

**How to test (optional):**
Register cv then go to /profil
### All of the following commands should pass.

- [x] `npm test`
- [x] `npm run lint`
- [ ] `npm run cypress`

### Browser testing

- [x] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] IE11
- [ ] Edge

### CODE REVIEW

File: src/views/CreateProfile/SaveCV.tsx [hunk 1]
```diff
@@ -43,17 +43,21 @@ export function removeTypename(prop: any): any {
 
 export const SAVE_CV = gql`
```
Comment (dubbelnisse) [inline src/views/CreateProfile/SaveCV.tsx]: Contact information is missing here.

File: src/views/Profile/Home.tsx [hunk 2]
```diff
@@ -1,15 +1,150 @@
 import React from 'react'
```
Comment (dubbelnisse) [inline src/views/Profile/Home.tsx]: Contact information is missing in this view. See Figma.

File: src/views/CreateProfile/SaveCV.tsx (hunk 1 above)
Comment (momentiris) [reply src/views/CreateProfile/SaveCV.tsx]: Yeah! Waiting until its supported on APi

File: src/views/Profile/Home.tsx (hunk 2 above)
Comment (momentiris) [reply src/views/Profile/Home.tsx]: Yep. Waiting until its on the api, but could add a placeholder

File: src/components/ProfileDataCard.tsx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  padding: ${({ theme }) => theme.space.small}px;
+  padding-bottom: 20px;
+  position: relative;
+  min-height: ${({ isTop }) => (isTop ? '100px' : '150px')};
+  flex: ${({ flexVal }) => (flexVal ? flexVal : 0.5)};
+`
+
+const ProfileDataCard: React.FC<IProfileDataCardProps> = ({
+  children,
+  flexVal = 0.5,
```
Comment (dubbelnisse) [inline src/components/ProfileDataCard.tsx]: Shouldn't this default to `1`. Egenskaper, Kompetens, Utbildning and Erfarenhet should take up equal space.

File: src/views/Profile/Home.tsx [hunk 4]
```diff
@@ -1,15 +1,150 @@
...
+                    key={education.id}
+                    mb="none"
+                    mt="none"
+                  >
+                    {education.programme}
+                  </Paragraph>
+                ))}
+            </ProfileDataCard>
+            <ProfileDataCard route="editplace">
+              {data.occupation && (
```
Comment (dubbelnisse) [inline src/views/Profile/Home.tsx]: The Erfarenhet heading is missing here.

File: src/views/Profile/Home.tsx [hunk 5]
```diff
@@ -1,15 +1,150 @@
...
+              isTop
+              noBackground
+              route="editplace"
+            >
+              <Paragraph fontSize="small" mb="none" mt="none">
+                {data.personalDescription}
+              </Paragraph>
+            </ProfileDataCard>
+            <ProfileDataCard flexVal={0.35} isImage isTop route="editplace">
+              Bild
```
Comment (dubbelnisse) [inline src/views/Profile/Home.tsx]: Remove the text `Bild`

File: src/views/Profile/Home.tsx [hunk 6]
```diff
@@ -1,15 +1,150 @@
...
+              flexVal={0.65}
+              isTop
+              noBackground
+              route="editplace"
+            >
+              <Paragraph fontSize="small" mb="none" mt="none">
+                {data.personalDescription}
+              </Paragraph>
+            </ProfileDataCard>
+            <ProfileDataCard flexVal={0.35} isImage isTop route="editplace">
```
Comment (dubbelnisse) [inline src/views/Profile/Home.tsx]: The image icon should be placed in the top right corner

File: src/views/CreateProfile/SaveCV.tsx (hunk 1 above)
Comment (dubbelnisse) [reply src/views/CreateProfile/SaveCV.tsx]: 👍

File: src/views/Profile/Home.tsx (hunk 2 above)
Comment (dubbelnisse) [reply src/views/Profile/Home.tsx]: 👍

File: src/components/ProfileDataCard.tsx (hunk 3 above)
Comment (momentiris) [reply src/components/ProfileDataCard.tsx]: true!

File: src/views/Profile/Home.tsx (hunk 4 above)
Comment (momentiris) [reply src/views/Profile/Home.tsx]: good catch

File: src/views/Profile/Home.tsx (hunk 5 above)
Comment (momentiris) [reply src/views/Profile/Home.tsx]: Yes please

File: src/views/Profile/Home.tsx (hunk 6 above)
Comment (momentiris) [reply src/views/Profile/Home.tsx]: true, fixed!

**Pull request merged**